ibis/backends/base/sql/alchemy/datatypes.py

```diff
@@ -367,6 +368,58 @@ def sa_struct(dialect, satype, nullable=True):
     return dt.Struct.from_tuples(pairs, nullable=nullable)
 
 
+try:
+    from snowflake.sqlalchemy import (
+        ARRAY,
+        OBJECT,
+        TIMESTAMP_LTZ,
+        TIMESTAMP_NTZ,
+        TIMESTAMP_TZ,
+        VARIANT,
+    )
+    from snowflake.sqlalchemy.snowdialect import SnowflakeDialect
+except ImportError:
+    pass
+else:
+
+    @dt.dtype.register(SnowflakeDialect, (TIMESTAMP_LTZ, TIMESTAMP_TZ))
+    def sa_sf_timestamp_ltz(dialect, satype, nullable=True):
```

Comment:
Could we place these into the snowflake backend?

Reply:
I think so, trying now.